src/middleware/authMiddleware.js

```diff
@@ -0,0 +1,17 @@
+import types from '../actions/actionTypes';
+
+const authMiddleware = () => next => (action) => {
+  if (action.type === `${types.LOGIN}_SUCCESS`) {
+    const { token } = action.payload.data.user;
+    action.payload.token = action.payload.data.user.token;
+    localStorage.setItem('token', token);
```

Comment:
Don't use`localStorage` to store user sensitive information like authentication `token`. It is not secure. You can see `https://dev.to/rdegges/please-stop-using-local-storage-1i04` for more info.

Reply:
Noted, I'll look into more options for storing the token.